tests/unittests/CMakeLists.txt

```diff
@@ -75,22 +75,19 @@ add_glow_test(ConvTest
               ${GLOW_BINARY_DIR}/tests/ConvTest)
 LIST(APPEND UNOPT_TESTS ./tests/ConvTest -optimize-ir=false &&)
 
-if(GLOW_WITH_CPU)
-  add_executable(CPUDeviceManagerTest
-                 CPUDeviceManagerTest.cpp)
-  target_link_libraries(CPUDeviceManagerTest
-                        PRIVATE
-                          Backends
-                          DeviceManager
-                          CPUDeviceManager
-                          Graph
-                          IR
-                          ExecutionEngine
-                          Optimizer
-                          gtest
-                          TestMain)
-  add_glow_test(CPUDeviceManagerTest ${GLOW_BINARY_DIR}/tests/CPUDeviceManagerTest --gtest_output=xml:CPUDeviceManagerTest.xml)
-endif()
+add_executable(DeviceManagerTest
+               DeviceManagerTest.cpp)
+target_link_libraries(DeviceManagerTest
```

Comment:
how do you test CPUDeviceManager here if there is no dependency on it?

Reply:
It is included in the DeviceManager target if GLOW_WITH_CPU is enabled